statsmodels/tsa/tsatools.py

```diff
@@ -652,6 +652,10 @@ def freq_to_period(freq):
         return 12
     elif freq == 'B' or freq == 'W' or freq.startswith('W-'):
         return 52
+    elif freq == 'D' or freq == 'C' or freq.startswith('D-'):
+        return 365
```

Comment:
hmmm I'll go with Hyndman on this and say I should change it to 7 my bad

http://robjhyndman.com/hyndsight/dailydata/